### DISCUSSION TITLE
How to extend the training set and begin from an already trained model

### DISCUSSION THREAD

**Discussion by DingChangjie** [deepmodeling/deepmd-kit#666] [Q&A] [answered]
Hi, I've tentatively trained a model with a relatively small training set, which only used a subset of my prepared AIMD data. What should I do if I want to extend my training set (so as to have those remaining AIMD trajectories used) and begin training with parameters inherited from the previous model ?

**Comment by njzjz** [comment, marked as answer]
See https://deepmd.readthedocs.io/en/v1.3.3/use-deepmd-kit.html?highlight=restart#training.

> --init-model model.ckpt, for example, initializes the model training with an existing model that is stored in the checkpoint model.ckpt, the network architectures should match.

**Reply by Sulesk1** [reply]
I appreciate your answer, could you please clarify why would someone use the init instead of the restart option and vice versa ? thank you.

**Reply by njzjz** [reply]
"init" would reset the index of steps.
